src/main/java/com/example/demo/assembler/UserAssembler.java

```diff
@@ -0,0 +1,66 @@
+package com.example.demo.assembler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+import com.example.demo.model.User;
+import com.example.demo.response.dto.UserDto;
+import com.example.demo.response.dto.UserResponseDto;
+
+@Component
+public class UserAssembler {
+	
+	public UserResponseDto toModel(List<User> users, Optional<User> opUser)
+	{
+		UserResponseDto userResponse = new UserResponseDto();
```

Comment:
We don't put any logic in assembler it just consists of mapping of entity

Reply:
Made another Assembler